tests/python/relay/test_ir_text_printer.py

```diff
@@ -46,6 +48,46 @@ def show(text):
         print(text)
 
 
+def get_darknet(model_name):
```

Comment:
BTW, we have a darknet parser. We may want to move the test there. I remember we have yolov3 tutorial as well. Should the same probably be present in the tutorial?

Reply:
I think the test makes more sense here, since we want to test darknet IRModule -> text -> IRModule, and this test file is solely dedicated to this pattern.

Could you elaborate on what you mean by "the same should probably be present in the tutorial"? You may be referring to this one: https://tvm.apache.org/docs/tutorials/frontend/from_darknet.html

Reply:
yeah, that is what I was referring to